engines/stark/gfx/driver.cpp

```diff
@@ -85,13 +85,27 @@ void Driver::computeScreenViewport() {
 	}
 }
 
-Common::Rect Driver::gameViewport() const {
-	Common::Rect game = Common::Rect(_screenViewport.width(), _screenViewport.height() * kGameViewportHeight / kOriginalHeight);
-	game.translate(_screenViewport.left, _screenViewport.top + _screenViewport.height() * kTopBorderHeight / kOriginalHeight);
-
+Common::Rect Driver::gameViewport(bool unscaled) const {
+	uint32 width = 640;
+	uint32 height = 480;
+	uint32 left = 0;
+	
+	if(unscaled){
```

Comment:
Please make sure to follow our code formatting conventions: http://wiki.scummvm.org/index.php/Code_Formatting_Conventions